lib/rubocop/cop/metrics/utils/abc_size_calculator.rb

```diff
@@ -22,21 +22,43 @@ module AbcSizeCalculator
           CONDITION_NODES = CyclomaticComplexity::COUNTED_NODES.freeze
 
           def self.calculate(node)
-            assignment = 0
-            branch = 0
-            condition = 0
+            AbcSizeCalculator.new(node).calculate
```

Comment:
Could you drop the class name here?
```suggestion
            new(node).calculate
```